artifacthub/designs/kubernetes_basic/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,23 @@
+version: 1.0.0
```

Comment:
I created this manually but ideally this file would be auto-generated. Maybe in some export process?

Reply:
You are so correct. Currently, Meshery Server does not automatically version designs, leaving this bit to users (for now). Automatic versioning is a desired feature.